.github/scripts/windows/find-target-branch.bat

```diff
@@ -3,6 +3,6 @@
 for /f "usebackq tokens=3" %%i in (`findstr PHP_MAJOR_VERSION main\php_version.h`) do set BRANCH=%%i
 for /f "usebackq tokens=3" %%i in (`findstr PHP_MINOR_VERSION main\php_version.h`) do set BRANCH=%BRANCH%.%%i
 
-if /i "%BRANCH%" equ "8.4" (
+if /i "%BRANCH%" equ "8.5" (
```

Comment:
Note that this would have solved the failing CI on Windows. I wasn't aware of this file.